### ISSUE TITLE
Position of variables is incorrectly parsed

### ISSUE THREAD

**Issue by armano2** [mysticatea/vue-eslint-parser#19]
Start and end pos in variables is moved by 1 character

27 - 32 is valid in this case, `VLiteral` and `HTMLLiteral` seems to contains `"`

template:
```html
<template><template scope="props"><span v-if="props"></span></template></template>
```

```json
"variables": [
  {
      "id": {
          "type": "Identifier",
          "start": 27,
          "end": 32,
          "loc": {
              "start": {
                  "line": 1,
                  "column": 27
              },
              "end": {
                  "line": 1,
                  "column": 32
              }
          },
          "range": [
              27,
              32
          ],
          "name": "props"
      },
      "kind": "scope"
  }
]
```

```json
{
  "type": "HTMLLiteral",
  "range": [
      26,
      33
  ],
  "loc": {
      "start": {
          "line": 1,
          "column": 26
      },
      "end": {
          "line": 1,
          "column": 33
      }
  },
  "value": "props"
},
```

```json
"value": {
    "type": "VLiteral",
    "range": [
        26,
        33
    ],
    "loc": {
        "start": {
            "line": 1,
            "column": 26
        },
        "end": {
            "line": 1,
            "column": 33
        }
    },
    "value": "props"
}
```

**Comment by mysticatea** [comment]
Thank you for this report.

However, this is intentional. 

- `HTMLLiteral` and `VLiteral` are similar to [`Literal` node](https://github.com/estree/estree/blob/master/es5.md#literal) of ESTree and `String` token. Those include quotes.
- The `scope` attribute is not a directive, so `VAttribute#value` is not `VExpressionContainer` (related to [docs/ast.md#VAttribute](https://github.com/mysticatea/vue-eslint-parser/blob/master/docs/ast.md#vattribute)).
- `Identifier` nodes express `IdentifierName` of ECMAScript, so it does not include quotes.
